docs/orchestration/index.md

```diff
@@ -233,103 +169,26 @@ stack {
 }
 ```
 
-The order of execution will be:
+With these settings, the execution sequence will be **stack-a** -> **stack-c** -> **stack-b**.
 
-* stack-a
-* stack-c
-* stack-b
-
-One example of terramate command that leverages order of
-execution is **terramate run**.
-
-This will run **terraform** plan on all stacks, but respecting ordering:
+One command that considers this execution order is `terramate run`, which runs `terraform plan` on all stacks in the defined sequence:
 
 ```sh
 terramate run terraform plan
 ```
 
-### Change Detection And Ordering
```

Comment:
hmm why do we delete all of this? Please explain :)

Reply:
I chose not to discuss this topic in the Orchestration Section because it will be fully discussed in the next section, which is about Change Detection. I did this to follow best practices. The aim was to concentrate on one idea and give a detailed guide on how to apply it on each page, instead of mixing up many ideas in one place.